styleguide/core/article/RichTextBody.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
I think this example should be hidden from the styleguide navigation. Would you prefix the filename with an `_`?